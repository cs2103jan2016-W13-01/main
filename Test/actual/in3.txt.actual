Task added successfully: task 1  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
Task added successfully: task 2  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
Task added successfully: task 3  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
5. incomplete; task 3; -; -; Once
Task added successfully: task 4  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
5. incomplete; task 3; -; -; Once
6. incomplete; task 4; -; -; Once
Task added successfully: task 5  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
5. incomplete; task 3; -; -; Once
6. incomplete; task 4; -; -; Once
7. incomplete; task 5; -; -; Once
Task added successfully: task 6  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
5. incomplete; task 3; -; -; Once
6. incomplete; task 4; -; -; Once
7. incomplete; task 5; -; -; Once
8. incomplete; task 6; -; -; Once
Task added successfully: task 7  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
5. incomplete; task 3; -; -; Once
6. incomplete; task 4; -; -; Once
7. incomplete; task 5; -; -; Once
8. incomplete; task 6; -; -; Once
9. incomplete; task 7; -; -; Once
Task added successfully: task 8  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
5. incomplete; task 3; -; -; Once
6. incomplete; task 4; -; -; Once
7. incomplete; task 5; -; -; Once
8. incomplete; task 6; -; -; Once
9. incomplete; task 7; -; -; Once
10. incomplete; task 8; -; -; Once
Task added successfully: task 9  Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; go to library; -; -; Once
3. incomplete; task 1; -; -; Once
4. incomplete; task 2; -; -; Once
5. incomplete; task 3; -; -; Once
6. incomplete; task 4; -; -; Once
7. incomplete; task 5; -; -; Once
8. incomplete; task 6; -; -; Once
9. incomplete; task 7; -; -; Once
10. incomplete; task 8; -; -; Once
11. incomplete; task 9; -; -; Once
Task added successfully: task 10 by 14:00 15 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; task 10; -; 14:00 15 Apr; Once
3. incomplete; go to library; -; -; Once
4. incomplete; task 1; -; -; Once
5. incomplete; task 2; -; -; Once
6. incomplete; task 3; -; -; Once
7. incomplete; task 4; -; -; Once
8. incomplete; task 5; -; -; Once
9. incomplete; task 6; -; -; Once
10. incomplete; task 7; -; -; Once
11. incomplete; task 8; -; -; Once
12. incomplete; task 9; -; -; Once
Task added successfully: task 11 from 10:50 28 Apr to 10:50 29 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; task 10; -; 14:00 15 Apr; Once
3. incomplete; task 11; 10:50 28 Apr; 10:50 29 Apr; Once
4. incomplete; go to library; -; -; Once
5. incomplete; task 1; -; -; Once
6. incomplete; task 2; -; -; Once
7. incomplete; task 3; -; -; Once
8. incomplete; task 4; -; -; Once
9. incomplete; task 5; -; -; Once
10. incomplete; task 6; -; -; Once
11. incomplete; task 7; -; -; Once
12. incomplete; task 8; -; -; Once
13. incomplete; task 9; -; -; Once
Task added successfully: task 12 by 15:00 27 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; task 10; -; 14:00 15 Apr; Once
3. incomplete; task 12; -; 15:00 27 Apr; Once
4. incomplete; task 11; 10:50 28 Apr; 10:50 29 Apr; Once
5. incomplete; go to library; -; -; Once
6. incomplete; task 1; -; -; Once
7. incomplete; task 2; -; -; Once
8. incomplete; task 3; -; -; Once
9. incomplete; task 4; -; -; Once
10. incomplete; task 5; -; -; Once
11. incomplete; task 6; -; -; Once
12. incomplete; task 7; -; -; Once
13. incomplete; task 8; -; -; Once
14. incomplete; task 9; -; -; Once
Task added successfully: task 13 from 15:00 26 Apr to 17:00 26 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; task 10; -; 14:00 15 Apr; Once
3. incomplete; task 13; 15:00 26 Apr; 17:00 26 Apr; Once
4. incomplete; task 12; -; 15:00 27 Apr; Once
5. incomplete; task 11; 10:50 28 Apr; 10:50 29 Apr; Once
6. incomplete; go to library; -; -; Once
7. incomplete; task 1; -; -; Once
8. incomplete; task 2; -; -; Once
9. incomplete; task 3; -; -; Once
10. incomplete; task 4; -; -; Once
11. incomplete; task 5; -; -; Once
12. incomplete; task 6; -; -; Once
13. incomplete; task 7; -; -; Once
14. incomplete; task 8; -; -; Once
15. incomplete; task 9; -; -; Once
Task added successfully: task 14 from 16:00 24 Apr to 18:00 25 Apr Once
1. incomplete; go to lecture; -; 10:00 05 Apr; Once
2. incomplete; task 10; -; 14:00 15 Apr; Once
3. incomplete; task 14; 16:00 24 Apr; 18:00 25 Apr; Once
4. incomplete; task 13; 15:00 26 Apr; 17:00 26 Apr; Once
5. incomplete; task 12; -; 15:00 27 Apr; Once
6. incomplete; task 11; 10:50 28 Apr; 10:50 29 Apr; Once
7. incomplete; go to library; -; -; Once
8. incomplete; task 1; -; -; Once
9. incomplete; task 2; -; -; Once
10. incomplete; task 3; -; -; Once
11. incomplete; task 4; -; -; Once
12. incomplete; task 5; -; -; Once
13. incomplete; task 6; -; -; Once
14. incomplete; task 7; -; -; Once
15. incomplete; task 8; -; -; Once
16. incomplete; task 9; -; -; Once
