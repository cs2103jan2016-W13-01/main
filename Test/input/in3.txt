add task 1
add task 2
add task 3
add task 4
add task 5
add task 6
add task 7
add task 8
add task 9
add task 10 2pm
add task 11 04/28/2016 04/29/2016
add task 12 3pm 04/27/2016
add task 13 3pm 5pm 04/26/2016
add task 14 4pm 6pm 04/24/2016 04/25/2016