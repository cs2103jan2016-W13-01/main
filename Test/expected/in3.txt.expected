Task added successfully
1. incomplete; task 1; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
3. incomplete; task 3; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
3. incomplete; task 3; -; -; -
4. incomplete; task 4; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
3. incomplete; task 3; -; -; -
4. incomplete; task 4; -; -; -
5. incomplete; task 5; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
3. incomplete; task 3; -; -; -
4. incomplete; task 4; -; -; -
5. incomplete; task 5; -; -; -
6. incomplete; task 6; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
3. incomplete; task 3; -; -; -
4. incomplete; task 4; -; -; -
5. incomplete; task 5; -; -; -
6. incomplete; task 6; -; -; -
7. incomplete; task 7; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
3. incomplete; task 3; -; -; -
4. incomplete; task 4; -; -; -
5. incomplete; task 5; -; -; -
6. incomplete; task 6; -; -; -
7. incomplete; task 7; -; -; -
8. incomplete; task 8; -; -; -
Task added successfully
1. incomplete; task 1; -; -; -
2. incomplete; task 2; -; -; -
3. incomplete; task 3; -; -; -
4. incomplete; task 4; -; -; -
5. incomplete; task 5; -; -; -
6. incomplete; task 6; -; -; -
7. incomplete; task 7; -; -; -
8. incomplete; task 8; -; -; -
9. incomplete; task 9; -; -; -
Task added successfully
1. incomplete; task 10; 06 Apr 14:00; -; -
2. incomplete; task 1; -; -; -
3. incomplete; task 2; -; -; -
4. incomplete; task 3; -; -; -
5. incomplete; task 4; -; -; -
6. incomplete; task 5; -; -; -
7. incomplete; task 6; -; -; -
8. incomplete; task 7; -; -; -
9. incomplete; task 8; -; -; -
10. incomplete; task 9; -; -; -
Task added successfully
1. incomplete; task 10; 06 Apr 14:00; -; -
2. incomplete; task 11; 28 Apr 23:12; 29 Apr 23:12; -
3. incomplete; task 1; -; -; -
4. incomplete; task 2; -; -; -
5. incomplete; task 3; -; -; -
6. incomplete; task 4; -; -; -
7. incomplete; task 5; -; -; -
8. incomplete; task 6; -; -; -
9. incomplete; task 7; -; -; -
10. incomplete; task 8; -; -; -
11. incomplete; task 9; -; -; -
Task added successfully
1. incomplete; task 10; 06 Apr 14:00; -; -
2. incomplete; task 12; 27 Apr 15:00; -; -
3. incomplete; task 11; 28 Apr 23:12; 29 Apr 23:12; -
4. incomplete; task 1; -; -; -
5. incomplete; task 2; -; -; -
6. incomplete; task 3; -; -; -
7. incomplete; task 4; -; -; -
8. incomplete; task 5; -; -; -
9. incomplete; task 6; -; -; -
10. incomplete; task 7; -; -; -
11. incomplete; task 8; -; -; -
12. incomplete; task 9; -; -; -
Task added successfully
1. incomplete; task 10; 06 Apr 14:00; -; -
2. incomplete; task 13; 26 Apr 15:00; 26 Apr 17:00; -
3. incomplete; task 12; 27 Apr 15:00; -; -
4. incomplete; task 11; 28 Apr 23:12; 29 Apr 23:12; -
5. incomplete; task 1; -; -; -
6. incomplete; task 2; -; -; -
7. incomplete; task 3; -; -; -
8. incomplete; task 4; -; -; -
9. incomplete; task 5; -; -; -
10. incomplete; task 6; -; -; -
11. incomplete; task 7; -; -; -
12. incomplete; task 8; -; -; -
13. incomplete; task 9; -; -; -
Task added successfully
1. incomplete; task 10; 06 Apr 14:00; -; -
2. incomplete; task 14; 24 Apr 16:00; 25 Apr 18:00; -
3. incomplete; task 13; 26 Apr 15:00; 26 Apr 17:00; -
4. incomplete; task 12; 27 Apr 15:00; -; -
5. incomplete; task 11; 28 Apr 23:12; 29 Apr 23:12; -
6. incomplete; task 1; -; -; -
7. incomplete; task 2; -; -; -
8. incomplete; task 3; -; -; -
9. incomplete; task 4; -; -; -
10. incomplete; task 5; -; -; -
11. incomplete; task 6; -; -; -
12. incomplete; task 7; -; -; -
13. incomplete; task 8; -; -; -
14. incomplete; task 9; -; -; -
